out/extension.js

```diff
@@ -28,6 +30,8 @@ let timeout = undefined;
 let diagnosticCollection = undefined;
 let specConfig = undefined;
 let snippets = {};
+let spl2Client = undefined;
```

Comment:
nit: not sure if we need to assign `undefined` because by default, `let spl2Client;` is undefined.

Reply:
I was following the convention in the file there already but yeah let me change them all